packages/react-split/src/index.js

```diff
@@ -65,7 +65,7 @@ class SplitWrapper extends React.Component {
         // Destroy and re-create split if options changed
         if (needsRecreate) {
             options.minSize = minSize
-            options.sizes = this.split.getSizes()
+            options.sizes = sizes
```

Comment:
`sizes` isn't a required prop so it could be undefined.

```suggestion
            options.sizes = sizes || this.split.getSizes()
```